### DISCUSSION TITLE
Unable to import boilerplate

### DISCUSSION THREAD

**Discussion by IonelPopJara** [DevSkillsHQ/discusssions#9] [Help] [answered]
Hi, I am trying to start my coding challenge but I ran into an issue.

These are the steps that I followed:

I'm running all the commands on a Windows computer.

1. Clone the task repo to my computer
2. Run the command  `set boilerplate=backend-boilerplate-dotnet` in the repo folder
3. Paste and run this code snippet:
```txt
cat <<EOF >>init.sh
#!/bin/sh
git clone https://github.com/DevSkillsHQ/$boilerplate.git
cp $boilerplate/package.json .
cp $boilerplate/cypress.json .
cp -r $boilerplate/app* .
rm -rf $boilerplate
npm i
git add cypress.json package* app*
git commit -m 'Init boilerplate'
EOF
chmod +x init.sh
./init.sh
rm init.sh
```
Step #2 seems to work fine but when I run the code snippet, I get a bunch of errors.

**Comment by IonelPopJara** [comment, marked as answer]
Turns out I had to run the snippet through `git bash` and use export instead of set.

**Reply by fenske** [reply]
Hi [@IonelPopJara](https://github.com/IonelPopJara). I'm Anton from DevSkills. Sorry to hear you experienced this problem, but I'm glad you figured it out.

If I understood correctly, you initially attempted to run the script from Powershell where it threw all those errors.

Nice workaround with git bash. I'll update the tutorial with this info.

I wish you the best of luck with the rest of your interview process!

**Reply by IonelPopJara** [reply]
Yes, that's exactly what happened.
Thank you!
